Sources/StreamChat/ChatClientFactory.swift

```diff
@@ -43,6 +43,7 @@ class ChatClientFactory {
 
     func makeApiClient(
         encoder: RequestEncoder,
+        databaseContainer: DatabaseContainer,
```

Comment:
We should avoid adding dependency to the db in the api client. Could this be a leftover actually?

Reply:
The DB is not part of the APIClient, actually, it is part of the StreamAttachmentUploader, which is a dependency of the APIClient. But the APIClient does not have a direct reference to the DB